src/App.js

```diff
@@ -433,7 +433,7 @@ class App extends Component {
                           <option value = '2020' >2020</option>
                           <option value = '2019'>2019</option>
                           <option value = '2018'>2018</option>
-                        </select>  {this.state.canClear === true ?   <FaRegTimesCircle onClick={this.clearFilter}  className='cancel-btn' value={theme === 'light' ? 'black': 'white'} style={{fontSize:'30px' , color : this.value  , cursor: 'pointer'} }/> : '' }
+                        </select>  {this.state.canClear === true ?   <FaRegTimesCircle onClick={this.clearFilter}  className='cancel-btn' value={theme === 'light' ? 'black': 'white'} style={{fontSize:'60px' , color : this.value  , cursor: 'pointer'} }/> : '' }
```

Comment:
Remove button, It's absolutely not necessary

Reply:
The button is to clear the searchFilter.

So one can return to the normal listing.

If the button is removed the user can't clear the searchFilter and can't view normal listings.

Reply:
> The button is to clear the searchFilter.
> 
> So one can return to the normal listing.
> 
> If the button is removed the user can't clear the searchFilter and can't view normal listings.

I know, implementation is great but it would have been better if we had a default `select` `option`, that takes it back to it's initial state.

Reply:
Alright

Reply:
> Alright

great work, can you also remove the info text, the `Option` will speak for itself.